crates/pathfinder/src/rpc/types.rs

```diff
@@ -477,13 +479,15 @@ pub mod reply {
     }
 
     /// L2 transaction as returned by the RPC API.
+    #[skip_serializing_none]
     #[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
     pub struct Transaction {
         pub txn_hash: StarknetTransactionHash,
+        /// Equal to `sender_address` for a Declare transaction
         pub contract_address: ContractAddress,
-        /// Absent for "deploy" transactions
+        /// Present only for an Invoke transaction
         pub entry_point_selector: Option<EntryPoint>,
-        /// Absent for "deploy" transactions
+        /// Present only for an Invoke transaction
```

Comment:
Pretty sure your other PR removed this comment.

Reply:
Yep, [these are the changes](https://github.com/eqlabs/pathfinder/pull/321/commits/051f16491e7ec50af8af0e9c2921fafbabb19bc2#diff-de073c53108c3c0e6543a72a3893ca2c6978b212062c620a4fb4ba2cccf882ddR484-R486) that you mentioned